views/index.html

```diff
@@ -113,7 +136,104 @@ <h2 class="card-title">Project name goes here</h2>
                 </section>
             </article>
         </section>
-        <section class="about intro">
+
+        <section id="Portfolio" class="projects">
+            <header>
+                <h1 class="name">
+                    Projects
+                </h1>
+                <img class="dots" src="../images/image_dots.svg" alt="footer image">
+                <div class="card-deskotop">
+                    <div class="img-size"></div>
+                    <h2 class="title">Project name goes here</h2>
+                    <ul class="languages">
+                        <li class="lang">HTML/CSS</li>
+                        <li class="lang">Ruby on Rails</li>
+                        <li class="lang">Javascript</li>
+                    </ul>
+                    <a href="#" class="project-link">See this project<img class="link-img" src="../images/Union2.png"
+                            alt="project link"></a>
+                </div>
+            </header>
+            <div class="projects2">
+                <img class="img-left" src="../images/Rectangle_46.svg" alt="project2">
+                <img src="../images/image-geometry_2.svg" alt="img project2">
+                <div class="card-deskotop">
+                    <div class="img2"></div>
+                    <h2 class="title">Project name goes...</h2>
+                    <ul class="languages">
+                        <li class="lang">HTML/CSS</li>
+                        <li class="lang">Ruby on Rails</li>
+                        <li class="lang">+1</li>
+                    </ul>
+                    <a href="#" class="project-link">See this project<img class="link-img" src="../images/Union2.png"
+                            alt="project link"></a>
+                    <div class="img3"></div>
+                </div>
+            </div>
+            <div class="projects3">
+                <img class="img4" src="../images/Rectangle_27.svg" alt="project3img">
+                <img class="img5" src="../images/image-geometry_3.svg" alt="project3img2">
+
+            </div>
```

Comment:
- [ ] It seems like there's a little problem with your images here, which is making your images overlap each other, so please make sure to fix that in order to stick to the given design as much as possible.
Yours:
![Captura de pantalla 2022-01-14 001638](https://user-images.githubusercontent.com/59100477/149424128-8193bb3d-4bbf-4b92-9200-e358020d5c6f.jpg)

Figma template:
![Captura de pantalla 2022-01-14 001654](https://user-images.githubusercontent.com/59100477/149424146-e4f7ad7c-bf64-4314-97d1-94444ea60061.jpg)

Note: Also, please make sure that once you fix this issue, the design is still following the Figma template in the rest of the document, as moving any piece may cause any kind of trouble later.